webpack/assets/javascripts/bundle.js

```diff
@@ -25,22 +25,9 @@ import * as configReportsModalDiff from './foreman_config_reports_modal_diff';
 import * as classEditor from './foreman_class_edit';
 import * as dashboard from './dashboard';
 import * as spice from './spice';
-import './bundle_datatables';
 import './bundle_lodash';
```

Comment:
Shouldn't this one also be moved to vendor?

Reply:
https://github.com/theforeman/foreman-js/pull/14

Reply:
theforeman/foreman-js#14 is closed.
This PR updated to allow full access to lodash using the window object.